docs/vault/overview.md

```diff
@@ -41,6 +41,12 @@ To support the integrity of the bridge, Vaults are also able to assume the role
 
 Head over to ["Installation"](/vault/installation) for a detailed setup guide.
 
+### Risks
+
+1. **Exchange Rate and Collateralization**: Vaults provide collateral to back locked BTC. If the collateralization falls below the liquidation collateral threshold, the Vault is liquidated. In case of a liquidation, Vaults collateral is seized as described [in the section on severe undercollateralization](vault/overview?id=severe-undercollateralization). Vaults with different collateral assets are [isolated](vault/overview?id=vault-isolation). This means that if, e.g., a Vault operator uses the same account id to run a DOT and USDC Vault, liquidating the DOT vault has not impact on the USDC Vault as long as the collateralization rate of the USDC Vault is below the liquidation threshold. [Learn how to maintain your collateralization here](vault/guide?id=managing-collateral).
```

Comment:
Should we mention the possibility of bugs? We'd mention that we are very careful, and that we have processes in place to minimize risks, including audits, but that with any piece of software there is a risk of bugs